ee/desktop/runner/runner.go

```diff
@@ -88,6 +88,32 @@ func InstanceDesktopProcessRecords() map[string]processRecord {
 	return instance.uidProcs
 }
 
+func InstanceDetectPresence(reason string, interval time.Duration) (bool, error) {
```

Comment:
singleton instance lazy route to avoid lots of plumbing ... maybe okay, possible revisit and do plumbing as time permits